.github/workflows/spellcheck.yml

```diff
@@ -1,4 +1,4 @@
-name: Spellcheck
+name: spellcheck
```

Comment:
is this a convention to name jobs lowercase?

Reply:
Not an official convention so much as to make it consistent across workflows. Right now there's a mixture of title case, lowercase, and kebab case in the current "Actions" view. If we want to converge on title casing in the future we can always throw a quick PR together.